src/components/shared/Accordion.js

```diff
@@ -69,7 +69,7 @@ class Accordion extends Component<Props, State> {
           onKeyDown={e => this.onHandleHeaderKeyDown(e, i)}
           onClick={() => this.handleHeaderClick(i)}
         >
-          <Svg name="arrow" className={opened ? "expanded" : ""} />
+          <button className={classnames("arrow", { expanded: opened })} />
```

Comment:
You need to add `import classnames from "classnames";` at the top of this file to use `classnames`

Reply:
thanks, I am new to all this sorry for the mistakes! will correct them immediately.